### DISCUSSION TITLE
[Question] Creating custom built-in connector for logic apps standard

### DISCUSSION THREAD

**Discussion by IvanTrashchenko** [Azure/logicapps#685] [Q&A] [answered]
Hi,

I am wondering, when creating custom built-in connector for logic apps standard, is there a way to acquire run id of current logic app using this connector in the code?

Thank you

**Comment by rarayudu** [comment, marked as answer]
Hi, you can use "@{workflow()['run']['name']}" expression to get the current run Id and then pass that in as an input to the built-in operation.

**Reply by IvanTrashchenko** [reply]
Thanks. Knew about this ability but wanted to know if there is some another solution without passing it as a parameter. 
Also, do you know, is this Built-in connector extensibility model supported in Azure? I created custom connector package and was able to use it in logic app in VS Code locally, but not after deployment 🤔

**Reply by rarayudu** [reply]
Yes, you should be able to use the custom connector package in Azure as well. Are there any errors you are facing?

**Reply by IvanTrashchenko** [reply]
I've managed to add my connector package via add-extension script locally:
![localdesigner](https://user-images.githubusercontent.com/36932736/205871150-f00525de-1532-47d3-ab22-ae680b3cb578.jpg)
Logic app succesfully runs:
![localsuccess](https://user-images.githubusercontent.com/36932736/205871263-5040c7f0-8b06-4fde-b9a1-bfaba8cad030.jpg)
After successful deployment of this logic app from VS Code this is what I see in portal:
![afterdepl](https://user-images.githubusercontent.com/36932736/205871371-0cc27f3d-9d94-4998-8af0-e32fa3217b75.jpg)
Important to note that connector package is deployed to Azure Artifacts feed.

**Reply by praveensri** [reply]
Can you please make sure the extension bundle is not used when you deploy to Azure, extension bundle id and version should not be there in the config?

![image](https://user-images.githubusercontent.com/52722196/215385416-d8ce7d0c-5564-4d8f-a9f2-442a9b50e326.png)

Please refer the comments under https://techcommunity.microsoft.com/t5/integrations-on-azure-blog/azure-logic-apps-running-anywhere-built-in-connector/ba-p/1921272

**Reply by IvanTrashchenko** [reply]
Thank you! It worked
